tests/namespaces.bats

```diff
@@ -373,3 +373,11 @@ general_namespace() {
 	[ "$status" -eq 0 ]
         expect_output "1:1"
 }
+
+@test "invalid userns-uid-map userns-gid-map" {
+	run_buildah 125 from --userns-uid-map 16  --userns-gid-map 0:48:16 scratch
+	expect_output 'error initializing ID mappings: userns-uid-map setting is malformed expected ["uint32:uint32:uint32"]: ["16"]'
+
+	run_buildah 125 from --userns-uid-map 0:32:16  --userns-gid-map 16 scratch
+	expect_output 'error initializing ID mappings: userns-gid-map setting is malformed expected ["uint32:uint32:uint32"]: ["16"]'
+}
```

Comment:
Could you add a test for regression purposes that covers the initial error?
```
# buildah bud --userns-gid-map a --userns-uid-map bogus .
error initializing ID mappings: userns-uid-map setting is malformed expected ["uint32:uint32:uint32"]: ["bogus"]
```
Otherwise this LGTM.